core/src/main/c/share/files.h

```diff
@@ -294,6 +294,8 @@ JNIEXPORT jboolean JNICALL Java_io_questdb_std_Files_setLastModified
  * Signature: (JJ)I
  */
 
+#define MAX_RW_COUNT 0x7ffff000
```

Comment:
nit: would be nice to add the following text from `man 2 read` as a clarifying comment for this constant:
```
On  Linux,  read() (and similar system calls) will transfer at most 0x7ffff000 (2,147,479,552) bytes, returning the number of bytes actually transferred.
```

Reply:
done